crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table.html

```diff
@@ -22,7 +22,11 @@
               <label class="usa-checkbox__label crt-checkbox__label checkbox-all" for="checkbox-all" aria-label="select all records"></label>
             </div>
           </th>
-          <th></th>
+          <th>
+            <a class="td-toggle-all display-flex" data-posted='false' href="#">
```

Comment:
Imran pointed out yesterday that one of the acceptance criteria was having it work well with a screen reader.

> Given I am on the view all table using a screen reader
when I hover over the carrot column select all checkbox
then the screen reader indicates ‘Expand all reports checkbox’

Should there be an aria-label on the anchor for this?

Reply:
So I just checked using NVDA (via Browserstack) and VoiceOver on Mac. Seems the alt text added for the arrow image indicates the functionality of the anchor link. I think putting an aria label would be redundant.

Reply:
I think the alt text satisfies the user requirement.